judge/bridge/django_handler.py

```diff
@@ -37,11 +37,11 @@ def on_submission(self, data):
         problem = data['problem-id']
         language = data['language']
         source = data['source']
-        judge_id = data['judge-id']
+        judge_ids = data['judge-ids']
```

Comment:
Thanks for the patch, I'll apply it to prod as-is.

I do have some concerns about naming here though, that I realized I had *after* merging the original patch, and now's a good time to address them.

The main one is that this isn't the judge *id*. It's the judge *name*, i.e. correct logic using this is `judge.name in judge_ids`, and *not* `judge.id in judge_ids` (the latter of which sounds more correct to a reader, but is actually wrong despite not raising any errors).

Reply:
I've renamed `judge_ids` to `judge_names` for specifying a judge to submit on. However, there are a lot of other places in the bridge which also uses `judge_id` when it should also be `judge_name`. Should those be renamed as well?